irods/session.py

```diff
@@ -25,7 +25,7 @@
 from irods.exception import (NetworkException, NotImplementedInIRODSServer)
 from irods.password_obfuscation import decode
 from irods import NATIVE_AUTH_SCHEME, PAM_AUTH_SCHEMES
-from . import DEFAULT_CONNECTION_TIMEOUT
+from . import MAXIMUM_CONNECTION_TIMEOUT, DEFAULT_CONNECTION_TIMEOUT
```

Comment:
let's switch the order here, alpha order AND the order they appear in `__init__.py`